src/common/pjrt_implementation/bindings.cc

```diff
@@ -0,0 +1,45 @@
+#include "common/pjrt_implementation/bindings.h"
```

Comment:
Separate main header of a .cc file with a blank line from other headers (in other files too)